.gitignore

```diff
@@ -1,5 +1,34 @@
-node_modules
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Hmm... the `yarn-error.log` appears to have crept into this PR. 🤔

Reply:
Oh, right - removed https://github.com/grafana/clock-panel/pull/110/commits/2b39f0a664cb97cbff0c16c0d87a3597b7eab641